src/Backend/Modules/FormBuilder/Layout/Templates/Add.html.twig

```diff
@@ -90,7 +90,7 @@
               <div class="col-md-12">
                 <label for="identifier" class="control-label">
                   {{ 'lbl.Identifier'|trans|ucfirst }}
-                  <abbr class="fa fa-info-circle" data-toggle="tooltip" title="{{ 'msg.HelpIdentifier'|trans }}"></abbr>
+                  <abbr class="fa fa-info-circle" aria-hidden="true" data-toggle="tooltip" title="{{ 'msg.HelpIdentifier'|trans }}"></abbr>
```

Comment:
Can you undo this? A better fix is available in https://github.com/forkcms/forkcms/pull/2320